### PR TITLE
verify-owners: add link on how to join the org

### PR DESCRIPTION
**Pull request by nikhita**
So that people can easily understand the process needed to become a member.

/sig contributor-experience
/cc @cblecker @spiffxp 
/assign @cblecker

### CODE REVIEW

**Comment by nikhita** [comment]
Ugh, right.

I'm not sure if it's worth adding a config option for it, but don't have strong opinions either way. ¯\\\_(ツ)\_/¯

**Comment by nikhita** [comment]
@cblecker thanks, updated.

**Comment by nikhita** [comment]
@cblecker ping for review :)

File: prow/plugins/verify-owners/verify-owners.go [hunk 1]
```diff
@@ -321,9 +321,9 @@ func handle(ghc githubClient, gc *git.Client, roc repoownersClient, log *logrus.
 
 		// prune old comments before adding a new one
 		cp.PruneComments(func(comment github.IssueComment) bool {
-			return strings.Contains(comment.Body, fmt.Sprintf(nonCollaboratorResponseFormat, ownersFileName, org))
+			return strings.Contains(comment.Body, fmt.Sprintf(nonCollaboratorResponseFormat, ownersFileName, org, triggerConfig.JoinOrgURL))
 		})
-		if err := ghc.CreateComment(org, repo, number, markdownFriendlyComment(org, nonTrustedUsers)); err != nil {
+		if err := ghc.CreateComment(org, repo, number, markdownFriendlyComment(org, triggerConfig.JoinOrgURL, nonTrustedUsers)); err != nil {
```
Comment (stevekuznetsov) [inline prow/plugins/verify-owners/verify-owners.go]: Almost feels like we should construct this with the github URL configured and org/repo?

Comment (stevekuznetsov) [reply prow/plugins/verify-owners/verify-owners.go]: Unless that link is a how-to, nevermind :man_facepalming:

**Pull request merged**